research/context/preselection/preselect_wikipedia_comments.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
```

Comment:
Make sure to include a license header in all .py files. And also a short outline of what the code in this file is intended to do. e.g. see: https://github.com/conversationai/conversationai-models/blob/master/experiments/tf_trainer/common/cnn_spec_parser.py

Reply:
Would be good to include in such a description what assumptions we are making about the shape of the data in the BigQuery table too.

Reply:
I added size of the table and assumptions w.r.t. the required table columns.